tip-wlan/templates/kafka-dependencies.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Namespace }}-common-kafka-certs
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  kafka-0.keystore.jks: {{ .Files.Get "resources/certs/kafka-server.pkcs12" | b64enc }}
+  kafka.truststore.jks: {{ .Files.Get "resources/certs/truststore.jks" | b64enc }}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Namespace }}-common-kafka-client-certs
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  client_keystore.jks: {{ .Files.Get "resources/certs/client_keystore.jks" | b64enc }}
+  kafka-server.pkcs12: {{ .Files.Get "resources/certs/kafka-server.pkcs12" | b64enc }}
+  truststore.jks: {{ .Files.Get "resources/certs/truststore.jks" | b64enc }}
+  server.pkcs12: {{ .Files.Get "resources/certs/server.pkcs12" | b64enc }}
+
+
+{{ if .Values.topics }}
+{{- $zk := printf "%s-wlan-zookeeper-headless" .Release.Namespace -}}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Namespace }}-wlan-kafka-config-{{ .Release.Revision }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 10
+  template:
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
+      - name: depends-on
+        image: confluentinc/cp-kafka:5.0.1
```

Comment:
@4c74356b41 should we use Bitnami's image here instead because we are deploying Bitnami Kafka

Reply:
@4c74356b41 do we want to parametrize the depends-on Docker image version and also the port number below (9092)